core/src/main/java/com/sequenceiq/distrox/v1/distrox/converter/InstanceTemplateParameterConverter.java

```diff
@@ -55,6 +55,7 @@ private AwsInstanceTemplateV4SpotParameters convert(AwsInstanceTemplateV1SpotPar
 
     private AwsEncryptionV4Parameters convert(AwsEncryptionV1Parameters source, DetailedEnvironmentResponse environment) {
         AwsEncryptionV4Parameters response = new AwsEncryptionV4Parameters();
+        String dataHubEncryptionKey = source.getKey();
         EncryptionType dataHubEncryptionKeyType = source.getType();
         if (EncryptionType.CUSTOM.equals(dataHubEncryptionKeyType)) {
             response.setKey(source.getKey());
```

Comment:
This applies to lines 61-62:
```
            response.setKey(source.getKey());
            response.setType(source.getType());
```

* `source.getKey()` can be replaced with `dataHubEncryptionKey`.
* `source.getType()` can be replaced with `dataHubEncryptionKeyType`.